includes/AssignPostTable.php

```diff
@@ -120,7 +122,7 @@
 		if($resCheck->num_rows == 0)
 		{
 			$stmt2 = $sqlConn->prepare("INSERT INTO $assignPostTable VALUES (?,?)");
-			$stmt2->bind_param ('ss', $row[0], $_POST[$row[0]] );
+			$stmt2->bind_param ('ss', $row[0], $_POST[$newRow] );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space before opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->